Doc/library/http.server.rst

```diff
@@ -51,9 +51,43 @@ handler.  Code to create and run the server looks like this::
    .. versionadded:: 3.7
 
 
-The :class:`HTTPServer` and :class:`ThreadingHTTPServer` must be given
-a *RequestHandlerClass* on instantiation, of which this module
-provides three different variants:
+.. class:: HTTPSServer(server_address, RequestHandlerClass,\
+                       bind_and_activate=True, *, certfile, keyfile=None,\
+                       password=None, alpn_protocols=None)
+
+   Subclass of :class:`HTTPServer` with a wrapped socket using the :mod:`ssl` module.
+   If the :mod:`ssl` module is not available, instantiating a :class:`!HTTPSServer`
+   object fails with a :exc:`RuntimeError`.
+
+   The *certfile* argument is the path to the SSL certificate chain file,
+   and the *keyfile* is the path to file containing the private key.
+
+   A *password* can be specified for files protected and wrapped with PKCS#8,
+   but beware that this could possibly expose hardcoded passwords in clear.
```

Comment:
Since it's for a testing server, users may not be experimented so maybe add a link to indicate that the parmeters certfile, keyfile and password are passed to https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_cert_chain.

Reply:
I think this is overcomplicating, specifying which functions the parameters are passed to.

Reply:
this is too detailed information that is usually found in the source code

Reply:
It's documented in `load_cert_chain` actually what `password` is expected to be

Reply:
It says:

> The password argument may be a function to call to get the password for decrypting the private key. It will only be called if the private key is encrypted and a password is necessary. It will be called with no arguments, and it should return a string, bytes, or bytearray. If the return value is a string it will be encoded as UTF-8 before using it to decrypt the key. Alternatively a string, bytes, or bytearray value may be supplied directly as the password argument. It will be ignored if the private key is not encrypted and no password is needed.

Reply:
You can just use `.. seealso::`

Reply:
i added it